src/app/components/hmcts-global-header/hmcts-global-header.component.html

```diff
@@ -57,17 +57,26 @@
     <div class="hmcts-primary-navigation__nav">
       <nav class="hmcts-primary-navigation transparent-background" aria-label="Primary navigation">
         <ul class="hmcts-primary-navigation__list">
-          <li class="hmcts-primary-navigation__item" *ngFor="let item of leftItems" >
+          <li class="hmcts-primary-navigation__item" *ngFor="let item of (leftItems | async)" >
             <a class="hmcts-primary-navigation__link" (click)="onEmitSubMenu(item)" [attr.aria-current]="item.active ? true: null" [routerLink]="item.href">{{item.text}}</a>
           </li>
         </ul>
       </nav>
     </div>
     <div class="hmcts-primary-navigation__search">
-      <!-- TODO: Should turn on and off -->
       <div class="hmcts-primary-navigation__item">
         <ul class="hmcts-primary-navigation__list">
-          <li class="hmcts-primary-navigation__item" *ngFor="let item of rightItems">
+          <li class="hmcts-primary-navigation__item" *ngFor="let item of (rightItems | async)">
```

Comment:
@johnbenjamin-hmcts - something has gone wrong with the merge of @ronaldmansveld's code from PR #1396; you have part of the change (the `| async` bit) on Line 69, followed by the old code on Lines 70-71, then Ronald's code from Lines 79-80 which is intended to replace Lines 69-71.
_Note also as part of Ronald's change, the `showFindCase` flag drops out of use._

Reply:
Hi @johnbenjamin-hmcts - following our discussion earlier this morning, I will fix this on the new `feature/global-search` branch. Once done, please rebase your PR against the branch and the issue should be resolved.